docs/terminology.md

```diff
@@ -0,0 +1,95 @@
+# Terminology
+
+## Overview
+This document contains all terminology used across compass documentation.
+
+## Management Plane 
+Abbr.: *MP*\
+\
+Includes *Compass*, *Runtime Provisioners*, *ServiceMesh* and *UI* components.
+
+The multi-tenant system which allows to:
+- create Applications
+- create Runtimes
+- manage Applications and Runtimes
+
+### MP Compass
+Abbr.: *Compass*\
+\
+Includes *Connector*, *Gateway*, *Director* and *Healtchecker* components.
+
+The multi-tenant system which allows to:
+- configure Applications
+- configure Runtimes
+- group Applications
+- assign Applications or Runtimes to the group
```

Comment:
Aren't this point and the previous one the same? I would remove the third point.